packages/wallet-adapter-core/src/types.ts

```diff
@@ -11,7 +11,8 @@ export type NetworkInfo = {
 
 export type AccountInfo = {
   address: string;
-  publicKey: string;
+  publicKey: string | string[];
+  minKeysRequired?: number
```

Comment:
I've had this question for a while but why do we need `minKeysRequired`?

Shouldn't that be coming from on-chain?

If you set minKeysRequired coming back from the wallet an attacker could easily just pass "1"

Reply:
Where can I get minKeysRequired on chain? As I know, there is an Auth Key only. According to https://aptos.dev/concepts/accounts#multisigner-authentication, developers can verify it by using `sha3-256(p_1 | . . . | p_n | K | 0x01)` and then comparing with Auth Key on chain.